tools/mitre/mitredb.py

```diff
@@ -168,6 +298,55 @@ def parse_table_(function, data_object):
 
     return table
 
+
+def parse_json_techniques(technique_json):
+    technique = Technique()
+
+    if technique_json.get(const.ID_t):
+        technique.id = technique_json[const.ID_t]
+    if technique_json.get(const.NAME_t):
+        technique.name = technique_json[const.NAME_t]
+    if technique_json.get(const.DESCRIPTION_t):
+        technique.description = technique_json[const.DESCRIPTION_t]
+    if technique_json.get(const.CREATED_j):
+        technique.created_time = datetime.strptime(technique_json[const.CREATED_j], const.TIME_FORMAT)
+    if technique_json.get(const.MODIFIED_j):
+        technique.modified_time = datetime.strptime(technique_json[const.MODIFIED_j], const.TIME_FORMAT)
+    if technique_json.get(const.MITRE_VERSION_j):
+        technique.mitre_version = technique_json[const.MITRE_VERSION_j]
+    if technique_json.get(const.MITRE_DETECTION_j):
+        technique.mitre_detection = technique_json[const.MITRE_DETECTION_j]
+    if technique_json.get(const.MITRE_NETWOR_REQ_j):
+        technique.network_requirements = technique_json[const.MITRE_NETWOR_REQ_j]
+    if technique_json.get(const.MITRE_REMOTE_SUPP_j):
+        technique.remote_support = technique_json[const.MITRE_REMOTE_SUPP_j]
+    if technique_json.get(const.REVOKED_BY_j):
+        technique.revoked_by = technique_json[const.REVOKED_BY_j]
+    if technique_json.get(const.DEPRECATED_j):
+        technique.deprecated = technique_json[const.DEPRECATED_j]
+    if technique_json.get(const.SUBTECHNIQUEOF_j):
```

Comment:
This is not OK. The `subtechnique-of` should be taken from the relationships. Please, remove this.